resources/dicts/relationship_events/normal_interactions/trust/increase.json

```diff
@@ -1,349 +1,400 @@
-            		"m_c is relieved when r_c says {PRONOUN/m_c/poss} cough is nothing to be worried about.",
-            		"m_c is relieved r_c doesn't judge {PRONOUN/m_c/object} for coming to {PRONOUN/r_c/object} with concerns that end up being nothing at all.",
-            		"m_c went to r_c genuinely concerned about a rough looking scratch only to be embarrassed and ashamed when r_c wipes it away. Turns out it was a very convincing smear of dried mud. r_c, amused by it all, promised not to tell anyone.",
-            		"r_c is extra careful treating m_c.",
-            		"m_c knows if anything goes wrong {PRONOUN/m_c/subject} can depend on r_c always."
-		],
-		"random_status_constraint": [
-			"medicine cat apprentice",
-			"medicine cat"
-		]
-	}
-]
+[
+	{
+		"id": "trust_inc_low1",
+		"intensity": "low",
+		"interactions": [
+			"m_c appreciates r_c telling {PRONOUN/m_c/object} that {PRONOUN/m_c/subject} had a feather stuck to {PRONOUN/m_c/poss} face.",
+			"r_c follows through on a small promise made with m_c.",
+			"r_c says something vulnerable to m_c.",
+			"m_c feels a little safer when r_c joins {PRONOUN/m_c/object} in the clearing.",
+			"m_c gets some good advice from r_c about something trivial.",
+			"r_c respects m_c's opinion despite disagreeing with it.",
+			"r_c vents about something trivial to m_c.",
+			"r_c sends a Clanmate to help m_c with something {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} struggling with.",
+			"r_c respects m_c's opinion despite disagreeing with it."
+		]
+	},
+	{
+		"id": "trust_inc_low2",
+		"intensity": "low",
+		"interactions": [
+			"m_c includes r_c in a conversation that r_c struggled to join.",
+			"r_c helps m_c hide from a Clanmate.",
+			"r_c makes a small sacrifice that benefits m_c.",
+			"r_c lets m_c know that a Clanmate was talking badly about {PRONOUN/m_c/object}.",
+			"r_c tells m_c that {PRONOUN/r_c/subject} was worried for {PRONOUN/m_c/object}.",
+			"r_c apologized to m_c for possibly hurting {PRONOUN/m_c/poss} feelings."
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_low3",
+		"intensity": "low",
+		"interactions": [
+			"m_c bestows wisdom onto r_c."
+		],
+		"reaction_random_cat": {
+			"trust": "increase",
+			"admiration": "increase"
+		},
+		"main_status_constraint": [
+			"elder"
+		]
+	},
+	{
+		"id": "trust_inc_med1",
+		"interactions": [
+			"m_c thinks about how r_c is always reliable.",
+			"m_c notices how r_c is being helpful around camp.",
+			"m_c has faith in r_c.",
+			"m_c notices that r_c has kept a secret of {PRONOUN/m_c/poss}s.",
+			"r_c seems respectful of m_c's boundaries as of late.",
+			"r_c explained something in detail to m_c that others wouldn't explain to {PRONOUN/m_c/object}.",
+			"m_c overhears r_c dispelling a false rumor about a Clanmate.",
+			"r_c helps m_c fix a matted piece of fur on {PRONOUN/m_c/poss} back.",
+			"m_c finds a Twoleg trinket. r_c agrees to keep it a secret.",

```

Comment:
this would benefit from a bit more detail! do they bring it back to camp? are they playing with it? just finding a trinket by itself isn't a huge deal after all

Reply:
Rewrote to this:
"m_c and r_c are playing with a fun Twoleg trinket they found.",

+ going to move it to a different file! Probably platonic increase.